src/utils.ts

```diff
@@ -8,6 +8,11 @@ import { Uri, workspace } from 'vscode';
 
 const textDecoder = new TextDecoder('utf-8');
 
+export interface CommandArgs {
+	commandPath: string;
+	args: string[];
+}
```

Comment:
It looks like you're using this to represent commands, but not all of the commands in this PR (e.g., the command you build in `runCargoKaniTest`). Again, we can leave this for a later PR but let's at least keep track of it.

Reply:
Yes, will do that. Thanks!